ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -270,6 +290,158 @@ module.exports = Backbone.AssociatedModel.extend({
     }
   },
 
+  convertLatLonLinePolyToUsng(points) {
```

Comment:
I would name each of these to `convertLatLonLinePolyTo[type]`. Maybe even rename the points params to latlonPoints.